src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -662,11 +661,47 @@ private void InitNativeProcess()
                 }
                 else
                 {
+#if UNIX
                     _isRunningInBackground = true;
                     if (!startInfo.UseShellExecute)
                     {
                         _isRunningInBackground = isWindowsApplication;
                     }
+#else
+                    // Notice about Windows Store applications.
```

Comment:
I don't think this is a store application specific point but any process that spawns another.  Any process could do this and `wt.exe` is just one of them while `skype.exe` stays alive as that's what they are programmed to do. I think this comment block should state why we get the exe this way as it's to let Windows resolve AppExecLinks rather than use undocumented structure fields.

Reply:
I think it makes sense to add info about AppX reparse point API gaps and maybe about our tracking issue. 
Usually @daxian-dbw comes with final great comment :-)

Reply:
Yea I think it's fine to mention this is for `AppExecLink` workaround but I think the extra info around wt and skype is just not needed, especially since the behaviour mentioned isn't actually related to AppExecLinks themselves. Usually just keep it concise is worthwhile IMO. But I'll leave it to @daxian-dbw to have the final say.

Reply:
> the behaviour mentioned isn't actually related to AppExecLinks

The whole workaround is needed only for Appx - other scenarios are covered by `IsWindowsApplication()` early.

Reply:
I understand, I just mean the whole `wt.exe` then calls `WindowsTerminal.exe` then exits and `skype.exe` just continues to run isn't something specific to AppExecLinks but just how those executables work. The AppExecLink behaviour not covered by `IsWindowsApplication()` is just how Windows interprets the reparse point itself and automatically repoints the final module to it's real target. I'm suggesting that the comment mentions that part rather than what's there currently.

Reply:
I updated the comment a bit.

Reply:
Looks great.